pandas/io/formats/style.py

```diff
@@ -416,16 +427,22 @@ def format_attr(pair):
             table_attributes=table_attr,
         )
 
-    def format(self, formatter, subset=None):
+    def format(self, formatter, subset=None, na_rep: Optional[str] = None):
```

Comment:
Not required here but if you wanted to in a follow up annotate other arguments would be super helpful